test/models/movie.test.js

```diff
@@ -13,10 +16,29 @@ describe('movie model', () => {
         'id',
         'title',
         'release_year',
+        'locations',
         'object'
       ]);
     });
 
+    it('serialzes with location', async () => {
+      const movie_payload = { name: 'Zodiac' };
+      const location_payload = { name: 'Bay Bridge' };
+
+      await Knex.raw('TRUNCATE movies CASCADE; TRUNCATE locations CASCADE; TRUNCATE locations_movies CASCADE;');
+
+      const movie = await new Movie().save(movie_payload);
+      const location = await new Location().save(location_payload);
+      const loc = await new Location({ id: location.id }).fetch();
```

Comment:
not entirely sure this part where we redefine `loc` is necessary (for serialization at least). I know that for certain controller tests, we usually create a resource then fetch it from the DB to ensure that there aren't any issues with inserting into the DB, but we can kinda separate and test that logic out in the controller instead.